src/include/platform/internal/GenericConfigurationManagerImpl.cpp

```diff
@@ -335,9 +334,11 @@ CHIP_ERROR GenericConfigurationManagerImpl<ConfigClass>::GetSpake2pIterationCoun
 template <class ConfigClass>
 CHIP_ERROR GenericConfigurationManagerImpl<ConfigClass>::GetSpake2pSalt(uint8_t * buf, size_t bufSize, size_t & saltLen)
 {
-    CHIP_ERROR err                                                   = CHIP_NO_ERROR;
-    char saltB64[BASE64_ENCODED_LEN(chip::kPBKDFMaximumSaltLen) + 1] = { 0 };
-    size_t saltB64Len                                                = 0;
+    // The maximum length of raw salt is chip::kPBKDFMaximumSaltLen (32 bytes).
+    static constexpr size_t kSpake2pSalt_MaxBase64Len = BASE64_ENCODED_LEN(32) + 1;
```

Comment:
Can we put this constant somewhere in the crypto headers?  Can everything involved include those?

Reply:
I've updated the fix and moved some of the Spake2p defines to the CHIPCryptoPAL.h